src/unix/linux-core.c

```diff
@@ -75,6 +75,16 @@
 # define CLOCK_BOOTTIME 7
 #endif
 
+/* Constant values for cgroups. */
+#define PROC_SELF_MOUNTINFO "/proc/self/mountinfo"
+#define PROC_SELF_CGROUP "/proc/self/cgroup"
+#define CGROUPS_VERSION_UNKNOWN 0x0
+#define CGROUPS_VERSION_1 0x1
+#define CGROUPS_VERSION_2 0x2
+#define CGROUPS_V2_CTRL_PREFIX "0::"
+#define CGROUPS_V2_CTRL_PREFIX_LEN 3
```

Comment:
Prefer `(sizeof(CGROUPS_V2_CTRL_PREFIX) - 1)`

Aside: these defines only seem to be used in one place so I'd be okay with open-coding them. That way a reader of the code doesn't have to look them up.